src/trading/ClaimProceeds.sol

```diff
@@ -0,0 +1,50 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/Controller.sol';
+import 'ROOT/reporting/Interfaces.sol';
+import 'ROOT/extensions/MarketFeeCalculator.sol';
+
+/**
+ * @title ClaimProceeds
+ * @dev This allows users to claim their money from a market by exchanging their shares 
+ */
+contract ClaimProceeds is Controlled {
+	uint256 private constant THREE_DAYS = 3 * 24 * 60 * 60;
+
```

Comment:
Can you please add a comment here (this is unrelated to your PR, but noticed in review):
```
AUDIT: Ensure that a malicious market can't subversively cause share tokens to be paid out incorrectly.
```